src/main/java/org/fusesource/jansi/io/AnsiToHtmlProcessor.java

```diff
@@ -0,0 +1,68 @@
+package org.fusesource.jansi.io;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+public class AnsiToHtmlProcessor extends AnsiProcessor {
+    private boolean concealOn = false;
+    private HtmlAnsiOutputStream haos;
```

Comment:
Why using another stream and not simply writing to the stream passed in the constructor ?
I don't think this `haos`  field is needed at all.

Reply:
So u mean like changing `AnsiToHtmlProcessor(OutputStream os) ` to `AnsiToHtmlProcessor(HtmlAnsiOutputStream os) `?
Because removing the haos completly would mean that this stuff in `processSetAttribute`:
```java
            case ATTRIBUTE_INTENSITY_BOLD:
                haos.writeAttribute("b");
                break;
            case ATTRIBUTE_INTENSITY_NORMAL:
                haos.closeAttributes();
                break;
            case ATTRIBUTE_UNDERLINE:
                haos.writeAttribute("u");
                break;
            case ATTRIBUTE_UNDERLINE_OFF:
                haos.closeAttributes();
                break;
```
would stop working.

Reply:
@gnodet Also changing it to `AnsiToHtmlProcessor(HtmlAnsiOutputStream os)` would mean that I would have to do this:
```java
public HtmlAnsiOutputStream(OutputStream os) {
        super(os,
                new WidthSupplier() {
                    @Override
                    public int getTerminalWidth() {
                        return Integer.MAX_VALUE;
                    }
                },
                AnsiMode.Default,
                new AnsiToHtmlProcessor(this),
...
``` 
And that wont work bc Im using `this` in the `super` constructor.

Reply:
I frankly don't see a way right now how that would work :/

Reply:
@gnodet Any ideas?